packages/pigeon/test/pigeon_lib_test.dart

```diff
@@ -156,9 +68,21 @@ void main() {
   });
 
   test('simple parse api', () {
-    final Pigeon dartle = Pigeon.setup();
-    final ParseResults parseResult = dartle.parseFile(thisPath,
-        types: <Type>[Api1], ignoresInvalidImports: true);
+    const String code = '''
+class Input1 {
+  String? input;
+}
+
+class Output1 {
+  String? output;
+}
+
+@HostApi()
+abstract class Api1 {
+  Output1 doit(Input1 input);
+}
+''';
+    final ParseResults parseResult = _parseSource(code);
```

Comment:
For all of these, consider inlining the string in the `_parseSource` call as the variable isn't actually used. Since there aren't any other arguments, the length of the inline argument isn't really an issue.